m_etcd/coordinator.go

```diff
@@ -319,7 +319,7 @@ startWatch:
 func (ec *EtcdCoordinator) parseTask(resp *etcd.Response) (task string, ok bool) {
 	// Sanity check / test path invariant
 	if !strings.HasPrefix(resp.Node.Key, ec.taskPath) {
-		metafora.Errorf("Received task from outside task path: %s", resp.Node.Key)
+		metafora.Errorf("%s received task from outside task path: %s", ec.name, resp.Node.Key)
```

Comment:
Yeah... I added a String method to the Coordinator in the last PR, but that seemed too random, so I renamed it to `Name()` (and use the underlying `name` field internally).

Any sort of contextual logger will need support from `logger.go`, and I just didn't get that far in this PR. Patches welcome! :D